sdk/avs/Azure.ResourceManager.Avs/src/Customized/CommonClusterProperties.cs

```diff
@@ -67,12 +67,16 @@ internal CommonClusterProperties(int? clusterSize, AvsPrivateCloudClusterProvisi
         }
 
         /// <summary> The cluster size. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public int? ClusterSize { get; set; }
         /// <summary> The state of the cluster provisioning. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public AvsPrivateCloudClusterProvisioningState? ProvisioningState { get; }
         /// <summary> The identity. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public int? ClusterId { get; }
         /// <summary> The hosts. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public IList<string> Hosts { get; }
```

Comment:
since these properties are still in use, we should not add the `EBNever` attribute to them.
Otherwise users will never see these properties even on an instance of `AvsManagementCluster`, which is not what we wpould like to see.